pyproject.toml

```
@@ -16,6 +16,7 @@ flake8-quotes = "*"
 flake8-coding = "*"
 flake8-commas = "*"
 flake8-import-order = "*"
+nose = "^1.3.7"
```

```suggestion
nose = "*"
```
Meine bitte an der Stelle wäre immer Sterne zu nutzen, weil ansonsten bei major releases man wird es einfach gar nicht mitkriegen, und vor allem bei dev-dependencies ist es mit Kompatibilität und Stabilität eher unbedenklich.